keps/sig-node/20190226-pod-overhead.md

```diff
@@ -98,152 +125,119 @@ Introduce a Pod.Spec.Resources field on the pod to specify the pods overhead.
 ```
 Pod {
   Spec PodSpec {
-    // Overhead is the resource overhead consumed by the Pod, not including
-    // container resource usage. Users should leave this field unset.
+    // Overhead is the resource overhead incurred from the runtime.
     // +optional
-    Overhead *ResourceRequirements
+    Overhead *Overhead
```

Comment:
Sorry if this wasn't what I said before, but I think the more complicated Overhead structure should only apply to the RuntimeClass. I think the PodSpec should still use the fixed ResourceRequirements for overhead, and the value calculated from the "overhead formula" during admission.

The only counter point I can think of is if in the future we wanted to account container overhead to each container cgroup... but I'm not sure how useful that is.

Reply:
this makes *much* more sense - not sure why I didn't pick that up before.  Agreed; just the single field makes sense imo.